robottelo/cli/factory.py

```diff
@@ -1033,6 +1035,121 @@ def make_filter(options=None):
+                                                               escaped with
+                                                               backslash
+         --name NAME                                           Policy name
+         --organization-ids ORGANIZATION_IDS                   REPLACE
+                                                               organizations
+                                                               with given ids.
+                                                               Comma separated
+                                                               list of values.
+                                                               containing comma
+                                                               should be quoted
+                                                               escaped with
+                                                               backslash
+         --organizations ORGANIZATION_NAMES                    Comma separated
+                                                               list of values.
+                                                               containing comma
+                                                               should be quoted
+                                                               escaped with
+                                                               backslash
+         --period PERIOD                                       Policy schedule
+                                                               period (weekly,
+                                                               monthly, custom)
+         --scap-content SCAP_CONTENT_TITLE                     SCAP content
+                                                               title
+         --scap-content-id SCAP_CONTENT_ID
+         --scap-content-profile-id SCAP_CONTENT_PROFILE_ID     Policy SCAP
+                                                               content
+                                                               profile ID
+         --tailoring-file TAILORING_FILE_NAME                  Tailoring file
+                                                               name
+         --tailoring-file-id TAILORING_FILE_ID
+         --tailoring-file-profile-id TAILORING_FILE_PROFILE_ID Tailoring file
+                                                               profile ID
+         --weekday WEEKDAY                                     Policy schedule
+                                                               weekday (only if
+                                                               period
+                                                               == “weekly”)
+         -h, --help                                            print help
+
+    """
+    # Assigning default values for attributes
+    # SCAP ID and SCAP profile ID is a required field.
+    if not options or not options.get('scap-content-id') or not options.get(
```

Comment:
according to your tests, all three parameters are mandatory, so please use `and` instead `or` or change all tests and remove unnecessary parameters